.changeset/brave-tomatoes-hunt.md

```diff
@@ -0,0 +1,5 @@
+---
+"@navikt/ds-react": patch
+---
+
+Date/MonthPicker: Upgrade react-day-picker to fix issue with React 19
```

Comment:
Ref. my previous comment, perhaps we should not mention react-day-picker either? Maybe just write "Fix issue with React 19"?

Reply:
I guess it's a transitive dependency, and consumers will be able to see it in their node_modules, but they should probably also not care what we use to build stuff under the hood? 🤔. it's just a patch, and I guess most will probably filter out release notes on the patch level anyways?. I say just keep it as this, it's at least transparent.